conda/recipes/libcudf/conda_build_config.yaml

```
@@ -22,11 +22,8 @@ gbench_version:
 gtest_version:
   - ">=1.13.0"
 
-aws_sdk_cpp_version:
-  - "<1.11"
-
-libarrow:
-  - "==12.0.0"
+libarrow_version:
+  - "=12"
```

The previous comment is a softer requirement -- but this one is a hard requirement. This must be 12.0.1 because of our CMake code searching for the same package version.
```suggestion
  - "==12.0.1.*"
```